src/bowling_SC.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class bowlingGame {
+
+  constructor() {
+    this.rolls = [[], [], [], [], [], [], [], [], [], []];
+    this.runningScore = 0;
+    this.frameIndex = 1;
```

Comment:
It's not obvious why the `frameIndex` begins at 1 - perhaps this is a good place to add a comment?